dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -531,26 +531,38 @@ git push --set-upstream origin "${branch}"
 
 Create a GitHub issue with the content generated via manual
 execution of the script below. You will use link to that issue in the next step. You need a GITHUB_TOKEN
-set as your environment variable.
-
-You can also pass the token as `--github-token` option in the script.
-You can also pass list of PR to be excluded from the issue with `--excluded-pr-list`.
+set as your environment variable or pass the token as `--github-token` option in the script.
```

Comment:
super nit: Will it be better if we provide the `export GITHUB_TOKEN=<your token>` here?

Reply:
most of times you don't need a token as the list is very small. for 80+ PRs in list it requires token

Reply:
> super nit: Will it be better if we provide the export GITHUB_TOKEN=<your token> here?

Both work. I personally use a token that I generate only for that and delete right after, so I prefer to use `--github-token` as it's single command to pass it in. But we can add example for both.

Reply:
> most of times you don't need a token as the list is very small. for 80+ PRs in list it requires token

My experience was that the rate limit hits me when It is about 95% done :D. But yeah we can mention that you might not needed. Also you need to remember that it very much depends where you are physically and how you are connected to the network. Github's (and any other) rate limiting on unauthenticated requests is based on the public IP address that you are connecting from. If you are behind a NAT where there are many other users using anonymous Github issue API calls, you share the rate limit with them - so for some users, the rate limit might start happening pretty much instanteneously - maybe you have your own public IP address at home and then you are fine, but if you are ina corporate network, then more often than not you might share your public IP address with 100s or even multiple thousands of people.  Using GITHUB_TOKEN is the only way to guarantee that a) your rate limit is way higher b) you actually have your own private rate limit.

So I personally ALWAYS use github token - and usually I just generate it for the time of running the command and discard right after - those are pretty much easily craated and I treat them as "one-time-use" only